scripts/export-bundles-sizes.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs')
+const path = require('path')
+const { getBrowserSdkVersion } = require('./lib/browser-sdk-version')
+const { getOrg2ApiKey } = require('./lib/secrets')
+const { runMain, fetch: fetchWrapper } = require('./lib/execution-utils')
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** any reason to not use the exported name directly?
‏
```suggestion
const { runMain, fetch } = require('./lib/execution-utils')
```

Reply:
I thought it would be clearer to specify the type of fetch.

Reply:
I think you're right It's a bit superfluous, I will change to only fetch